src/structures/Widget.js

```diff
@@ -57,7 +57,7 @@ class Widget extends Base {
      */
     this.members = new Collection();
     for (const member of data.members) {
-      this.members.set(member.id, new WidgetMember(this.client, member));
+      this.members.set(parseInt(member.id), new WidgetMember(this.client, member));
```

Comment:
You can keep this as a string! Sorry, was mostly a question about it 👍

Reply:
Same here x)